### PR TITLE
libdbusmenu-qt: Add homepage

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

Add `homepage` key to `package.yml` (Part of getsolus/packages#411)

**Test Plan**

Verified the added homepage loads information about the package

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/l/libdbusmenu-qt/pspec_x86_64.xml [hunk 1]
```diff
@@ -30,18 +31,11 @@
...
-            <Dependency release="2">libdbusmenu-qt</Dependency>
+            <Dependency release="3">libdbusmenu-qt</Dependency>
         </RuntimeDependencies>
         <Files>
-            <Path fileType="header">/usr/include/dbusmenu-qt5/CMakeFiles/Export/lib64/cmake/dbusmenu-qt5</Path>
-            <Path fileType="header">/usr/include/dbusmenu-qt5/CMakeFiles/dbusmenu-qt5.dir/dbusmenu-qt5_autogen</Path>
-            <Path fileType="header">/usr/include/dbusmenu-qt5/CMakeFiles/dbusmenu-qt5_autogen.dir</Path>
-            <Path fileType="header">/usr/include/dbusmenu-qt5/dbusmenu-qt5_autogen/include</Path>
-            <Path fileType="header">/usr/include/dbusmenu-qt5/dbusmenu-qt5_autogen/moc_predefs.h</Path>
-            <Path fileType="header">/usr/include/dbusmenu-qt5/dbusmenu_config.h</Path>
```
Comment (malfisya) [inline packages/l/libdbusmenu-qt/pspec_x86_64.xml:41]: Some files are gone. I don't understand why

File: packages/l/libdbusmenu-qt/pspec_x86_64.xml [hunk 2]
```diff
@@ -30,18 +31,11 @@
...
             <Path fileType="header">/usr/include/dbusmenu-qt5/dbusmenu_export.h</Path>
             <Path fileType="header">/usr/include/dbusmenu-qt5/dbusmenu_version.h</Path>
-            <Path fileType="header">/usr/include/dbusmenu-qt5/dbusmenuadaptor.h</Path>
```
Comment (malfisya) [inline packages/l/libdbusmenu-qt/pspec_x86_64.xml:44]: This too.

**Comment by malfisya** [comment]
So, some files are gone if I switch to new macro. With old macro all files are still there with only one exception, one file is replaced by some long string of numbers.

**Review by algent-al — APPROVED** [review]
LGTM, thanks!

**Pull request merged**